beanstalk_worker/__init__.py

```diff
@@ -56,6 +56,7 @@ def wrapper(*args, **kwargs):
                 task = Task()
                 user = kwargs.pop("user")
                 task.account = user.iaso_profile.account
+                task.created_by = user
                 task.launcher = user
```

Comment:
What I understand is that we assume that the creator and the launcher are always the same person. Is this correct?

Reply:
This is when the task is being created, so here indeed the creator and launcher will be the same person.

It's because of the "relaunch" functionality that I added the creator. When relaunching, only the launcher is updated (see https://github.com/BLSQ/iaso/blob/main/iaso/api/tasks/__init__.py#L135)